tasktiger/worker.py

```diff
@@ -764,10 +764,13 @@ def _finish_task_processing(self, queue, task, success):
         """
         log = self.log.bind(queue=queue, task_id=task.id)
 
+        when = time.time()
+        execution_duration = task.ts and when - task.ts
+
         def _mark_done():
             # Remove the task from active queue
             task._move(from_state=ACTIVE)
-            log.info('done')
+            log.info('done', execution_duration=execution_duration)
```

Comment:
I'm sure there might be an even better name, but would `processing_duration` be more accurate? I'd expect `execution_duration` to be the duration that it takes to actually execute the task, i.e. if it was measured somewhere within `_execute`, and not what we're actually doing here.